util/pytest_log_filter.py

```diff
@@ -0,0 +1,16 @@
+import re
+import sys
```

Comment:
Please add a comment explaining briefly what this program does and how to call it from the command line.